app/src/main/java/com/github/polybooks/FilteringBooksActivity.kt

```diff
@@ -142,6 +143,8 @@ class FilteringBooksActivity: FilteringActivity() {
         setupNavbar(findViewById(R.id.bottom_navigation), this, R.id.books, navBarListener)
     }
 
```

Comment:
I think you should just remove if not needed anymore

Reply:
Done